app/views/runs/stats/index.slim

```diff
@@ -19,101 +21,102 @@ article#stats
-              span Total potential timesave
-              small.tip-html< title="<code>PB - sum of best</code>" = "(?)"
-            td = format_milliseconds(@run.duration_ms(timing) - @run.sum_of_best_ms(timing))
-        tr
-          td
-            span Attempt
-          td = @run.attempts || '-'
-        tr
-          td
-            span Splits
-          td = @run.segments.size
-        tr
-          td
-            span Missed splits this run
-          td = @run.skipped_splits(timing).size
-        tr
-          td
-            span Exact run duration
-          td #{@run.duration_ms(timing).to_f / 1000}s
-        tr
-          td
-            span Shortest segment
-          td
-            - if @run.collapsed_segments(timing).size > 0
-              | #{@run.shortest_segment(timing).name} (
-              span = format_milliseconds(@run.shortest_segment(timing).duration_ms(timing))
-              | )
-            - else
-              span.time -
-        tr
-          td
-            span Longest segment
-          td
-            - if @run.longest_segment(timing).present?
-              | #{@run.longest_segment(timing).name} (
-              span = format_milliseconds(@run.longest_segment(timing).duration_ms(timing))
-              | )
-            - else
-              span.time -
-        tr
-          td
-            span Mean segment length
-          td
-            - if @run.collapsed_segments(timing).size > 0
-              span = format_milliseconds(@run.duration_ms(timing) / @run.collapsed_segments(timing).size)
-            - else
-              span.time -
-        tr
-          td
-            span Median segment length
-          td
-            - if @run.collapsed_segments(timing).size > 0
-              span = format_milliseconds(@run.median_segment_duration_ms(timing))
-            - else
-              span.time -
-        tr
-          td
-            span Total Playtime
-          td
-            - if @run.total_playtime_ms(timing) != 0
-              span = format_milliseconds(@run.total_playtime_ms(timing))
-            - else
-              span.time -
+              span Uploaded
+            td title=@run.created_at = "#{time_ago_in_words(@run.created_at)} ago"
+          - has_sob = @run.sum_of_best_ms(timing).present? && @run.sum_of_best_ms(timing) != 0
+          - has_attempts = @run.duration_ms(timing).present? && @run.duration_ms(timing) != 0
+          - if !has_sob
+            tr.tip title="This run has no gold splits recorded"
+              td
+                span Sum of best
+              td -
+          - else
+            tr
+              td
+                span Sum of best
+              td = format_ms(@run.sum_of_best_ms(timing))
+          - if !has_sob || !has_attempts
+            tr.tip title="This run has no gold splits recorded"
+              td
+                span Total potential timesave
+                small.tip-right< title="<code>PB - sum of best</code>" = "(?)"
```

Comment:
Since all the rest of the tips are FA icons, it might be worth considering turning this into one as well to make it consistent, but up to you on that one.